test/test_kramdown.rb

```diff
@@ -30,6 +30,24 @@ class TestKramdown < JekyllUnitTest
       assert_equal "<h1>Some Header</h1>", @markdown.convert('# Some Header #').strip
     end
 
+    context "when asked to disable the highlighter" do
+      should "accept nil" do
+        override = {
+          "kramdown" => {
+            "syntax_highlighter" => nil
+          }
+        }
+
+        markdown = Converters::Markdown.new(Utils.deep_merge_hashes(
+          @config, override
+        ))
+
+        result = nokogiri_fragment(markdown.convert "```ruby\nhello\n```")
+        select = "pre/code[contains(@class, 'language-ruby')][contains(text(),'hello')]"
+        refute result.xpath(select).empty?
```

Comment:
Doesn't look like this refutation is passing: https://travis-ci.org/jekyll/jekyll/jobs/107765459#L345